x/session/keeper/session_hydrator.go

```diff
@@ -243,8 +243,24 @@ func sha3Hash(bz []byte) []byte {
 	return hasher.Sum(nil)
 }
 
+// GetSessionStartBlockHeight returns the block height at which the session starts
+func GetSessionStartBlockHeight(blockHeight int64) int64 {
```

Comment:
Interesting... I added them as helper functions to factor these calculations since they are needed in contexts other than this keeper (mainly `testutils`) I wonder if we're not going to end-up passing the `NumBlocksPerSession` to these functions :thinking: